omarjabari/task_service.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, List
+from uuid import UUID
+from models import Task, UrgentTask
+
+
+class task_service:
+    def __init__(self):
+        self.tasks: Dict[UUID, Task] = {}
+
+    def create_task(self, task: Task):
+        if task.task_id in self.tasks:
+            raise ValueError("Task ID already exists")
+        self.tasks[task.task_id] = task
+        return task
+
+    def update_task(self, task_id: UUID, updated_task: Task):
```

Comment:
We need the `update_task` to have the ability to change a specific property (maybe one, maybe more than one) in the task object, so please edit this code to meet this requirement